package.json

```diff
@@ -308,14 +309,14 @@
     "reselect": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "ra-core": "^2.6.4",
-    "redux-form": "~7.0.0",
-    "styled-components": "^4.0.0",
-    "redux": "~3.0.0",
-    "react-redux": "~5.0.0",
-    "@bootstrap-styled/redux": "^1.0.0"
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "ra-core": "^2.8.5",
+    "redux-form": "~7.4.0",
+    "styled-components": "^4.1.3",
+    "redux": "~3.7.2",
```

Comment:
Hmm I know this is ra-core versions but we might still want a wider range, 3.0.0 for redux, 4.0.0 for styled etc... so far this is the least worst of the worst decision, this can be done by unselecting the peer with ncu , I am still looking for the proper command that should be doing that and write to the package.json properly. it should be `ncu --deps dev,prod -u` but I am not sure if that is working.